modules/site-health/audit-autoloaded-options/load.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Module Name: Audit Autoloaded Options
+ * Description: Adds Autoloaded options checker in Site Health checks.
+ * Experimental: No
```

Comment:
If we merge it like this, I'd be in favor of marking it as experimental based on my previous comment that this isn't reasonably actionable for end users.